src/core.js

```diff
@@ -1,4 +1,4 @@
-import { window, setTimeout, console } from "./globals";
+import { window, setTimeout, console, sessionStorage } from "./globals";
```

Comment:
> Do you think that should instead be a followup PR?

I always prefer followup PRs, more simple to review, and less responsibility on each single merges.

This other PR should also include docs.